src/renderer/components/reader/ReaderHeader.tsx

```diff
@@ -54,6 +54,8 @@ interface Props extends TranslatorProps {
 export class ReaderHeader extends React.Component<Props, undefined> {
     private enableFullscreenRef: any;
     private disableFullscreenRef: any;
+    private catchSettingFocus: any;
```

Comment:
For consistency with `enableFullscreenRef` and `disableFullscreenRef`, I suggest `catchSettingFocus` and `catchMenuFocus` are renamed with the `xxRef` suffix naming convention. For example: `navigationMenuButtonRef` and `settingsMenuButtonRef`. Thoughts?

Reply:
yes it's more relevant so i changed  my variables name